tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py35,py36,py37}-django{22},{py36,py37,py38,py39}-django{30,31,32},lint
+envlist = {py35,py36,py37}-django{22},{py36,py37,py38,py39}-django{30,31,32},{py38,py39,py310}-django{32,40},lint
```

Comment:
I think this should be simplified a bit, because the environment for Django 3.2 is tested twice with `py38` and `py39`.